src/ci/docker/x86_64-gnu-llvm-5.0/Dockerfile

```diff
@@ -10,11 +10,21 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   git \
   cmake \
   sudo \
-  gdb \
+  wget \
   llvm-5.0-tools \
   libedit-dev \
   zlib1g-dev \
-  xz-utils
+  xz-utils \
+  texinfo
+
+RUN wget http://ftp.gnu.org/gnu/gdb/gdb-8.1.1.tar.gz > /dev/null && \
+  tar -xzvf gdb-8.1.1.tar.gz > /dev/null && \
```

Comment:
FWIW, gdb 8.2 was released yesterday: http://lists.gnu.org/archive/html/info-gnu/2018-09/msg00001.html

Reply:
... forgot to mention, it includes some Rust fixes; if your patch goes in I'm hoping to use this when testing a patch I am working on.

Reply:
Good news! I try gdb8.2.